server/base/src/main/java/org/apache/accumulo/server/compaction/CompactionJobGenerator.java

```diff
@@ -65,7 +66,8 @@ public CompactionJobGenerator(PluginEnvironment env,
     serviceIds = servicesConfig.getPlanners().keySet().stream().map(CompactionServiceId::of)
         .collect(Collectors.toUnmodifiableSet());
 
-    dispatchers = Caffeine.newBuilder().maximumSize(10).build();
+    dispatchers = Caches.getInstance().createNewBuilder(CacheName.COMPACTION_DISPATCHERS, true)
```

Comment:
This cache is short lived, maybe it should not emit metrics?  Also there could be multiple instance of this object in multiple processes (like the manager and tserver via TabletMgtmIter), will that cause problems for metrics?

Reply:
Disabled metrics on this since you said it's short-lived, in 8bee224